### PR TITLE
Feat/affiliate apply

### LINKED ISSUE #305: Wallet authentication to apply as an affiliate
**Is your feature request related to a problem? Please describe.**
There is currently no need to authenticate with wallet to apply as an affiliate

**Describe the solution you'd like**
It would be better if users need to authenticate with their wallets to apply as an affiliate

### LINKED ISSUE #320: Frontend KYC validation for affiliate application
Users can apply as an affiliate only if they have completed KYC.

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Resolve #305 
- Resolve #320

### Steps to test

1. make run

#### CheckList

- [ ] Check you can't APPLY NOW until you are logged
- [ ] Check you can't APPLY NOW until you have:
  - Account is not already an affiliate
  - Account is not on the waiting list
  - Account does have KYC

### CODE REVIEW

File: webapp/src/utils/affiliate.js [hunk 1]
```diff
@@ -26,6 +26,10 @@ const REFERRAL_STATUS_IDS = {
   PAID: 6
 }
 const GUEST_ROLE = 'NON-AFFILIATED'
+const JOIN_REQUEST_STATUS = {
```
Comment (leisterfrancisco) [inline webapp/src/utils/affiliate.js:29]: @adriexnet @tetogomez could you confirm it is in the right place, please.

**Review by tetogomez — APPROVED** [review]
LGTM

File: webapp/src/routes/Home/index.js [hunk 2]
```diff
@@ -108,7 +104,7 @@ const Home = () => {
       await addJoinRequest({
         variables: {
           user: {
-            account,
+            account: account,
```
Comment (adriexnet) [inline webapp/src/routes/Home/index.js:107]: @leisterfrancisco just to know, why are you changing this from the short way `account` to `account: account,`?

**Review by adriexnet — APPROVED** [review]

File: webapp/src/routes/Home/index.js (hunk 2 above)
Comment (leisterfrancisco) [reply webapp/src/routes/Home/index.js:107]: I had fixed it but assume in a ```
ctrl + z
``` shortcut i got it back

Comment (leisterfrancisco) [reply webapp/src/routes/Home/index.js:107]: Very nice catch

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**